src/firefly/java/edu/caltech/ipac/firefly/server/cache/EhcacheProvider.java

```diff
@@ -65,7 +66,14 @@
             File ignoreSizeOf = getConfFile("ignore_sizeof.txt");
             System.setProperty("net.sf.ehcache.sizeof.filter", ignoreSizeOf.getAbsolutePath());
 
-            sharedManager = CacheManager.create(sharedConfig.getAbsolutePath());
+            try {
+                System.setProperty("net.sf.ehcache.sizeofengine.shared.VIS_SHARED_MEM",
+                                   "edu.caltech.ipac.firefly.server.cache.ObjectSizeEngineWrapper");
```

Comment:
I think using ObjectSizeEngineWrapper.class.getName() here would be even cleaner.  It will fail on setProperty so clearing it is not necessary.  It'll also help find/refactor class when needed.